deploy/helm/prometheus-overrides.yaml

```diff
@@ -19,28 +19,39 @@ prometheusOperator:
     enabled: false
 prometheus:
   additionalServiceMonitors:
-  - name: collection-sumologic
+  - name: collection-sumologic-fluentd
```

Comment:
I think we should explicitly call out logs, metrics and eventually traces rather than just fluentd alone.

Reply:
Which will require an update to fluent-bit output i believe.